src/cli/chem-comp-bond/create-table.ts

```diff
@@ -23,6 +23,7 @@ import { CCD_Schema } from '../../mol-io/reader/cif/schema/ccd';
 import { SetUtils } from '../../mol-util/set';
 import { DefaultMap } from '../../mol-util/map';
 import { mmCIF_chemCompBond_schema } from '../../mol-io/reader/cif/schema/mmcif-extras';
+import { ccd_chemCompAtom_schema } from '../../mol-io/reader/cif/schema/ccd-extras';
```

Comment:
With the added functionality, could you rename the chem-comp-bond cli tool to chem-comp-dict?

Reply:
Done in 3aa9ef7